### PR TITLE
Data file to Qt resource file system

### PR DESCRIPTION
**Pull request by ChrisRzech**
### Key features
* Initial data files are now static in Qt resource file system
* Upon program execution, resource data files are copied to the executable's folder so files can be edited and persistant between executions
* Database now loads data once per program execution

### Future Improvements
* none

### Notes
Permissions were tricky, decided to use `0777` for allowing all permissions just in case.

### CODE REVIEW

**Review by yosmo78 — APPROVED** [review]
AMEN!

File: src/main.cpp [hunk 1]
```diff
@@ -12,6 +14,20 @@ int main(int argc, char *argv[])
...
     MainWindow* window = nullptr;
 
+    /* Copy the resource files into the filesystem */
+    QFile(":/res/MLBInformation.csv").copy("MLBInformation.csv");
+    QFile(":/res/DistanceBetweenStadiums.csv").copy("DistanceBetweenStadiums.csv");
+
+    /* Set permissions for the new files */
+    QFile mlbInfo("MLBInformation.csv");
+    QFile distance("DistanceBetweenStadiums.csv");
+    mlbInfo.setPermissions(QFileDevice::Permissions(0777));
```
Comment (typokign) [inline src/main.cpp]: reasoning behind 777?

Comment (ChrisRzech) [reply src/main.cpp]: We gave up

Comment (typokign) [reply src/main.cpp]: but why execute?

Comment (yosmo78) [reply src/main.cpp]: self modifying code duh

Comment (ChrisRzech) [reply src/main.cpp]: yeah we writing AI on this project

File: src/main.cpp [hunk 2]
```diff
@@ -21,8 +22,8 @@ int main(int argc, char *argv[])
     /* Set permissions for the new files */
     QFile mlbInfo("MLBInformation.csv");
     QFile distance("DistanceBetweenStadiums.csv");
-    mlbInfo.setPermissions(QFileDevice::Permissions(0777));
-    distance.setPermissions(QFileDevice::Permissions(0777));
+    mlbInfo.setPermissions(QFileDevice::Permissions(07777));
```
Comment (typokign) [inline src/main.cpp]: ew, these are still not executables, why do they have +x, uid, gid, and sticky bit?

**Review by Roger-Chavez — APPROVED** [review]

**Pull request merged**